### DISCUSSION TITLE
How to increase the amount of connections allowed per node?

### DISCUSSION THREAD

**Discussion by razb-viola** [nodejs/help#4806] [Q&A]
I used autocannon in order to burst my api. I am trying to know how much connections are allowed to be opened simultaneously. 
So far, if I create up to 300 concurrent connections, I can burst my API with 300k requests and all of them will be fulfilled. 
If I increase it to 400-500 some requests will failed with ECONNECT -54 error.
The more concurrent connections I try to overload on the node is the more errors will be.

I know that in order to scale I should provision more nodes. But I am just researching and trying to understand how much connections, just one node, is practically can handle. And, if I can configure this my self - not that I want - just wanna find the limits as a research.

Thanks.

**Comment by theanarkh** [comment]
I think you can refer to `backlog` parameter in [listen](https://nodejs.org/docs/latest/api/net.html#serverlisten) API.

**Reply by razb-viola** [reply]
Could you please write an example of how to utilize it the way I want? I tried to figure this out from the docs, but with no clue.

**Reply by marco-ippolito** [reply]
check https://nodejs.org/api/net.html#serverlisten `backlog` option
